include/denovo_discovery/discover_main.h

```diff
@@ -29,7 +29,11 @@ struct DiscoverOptions {
     bool illumina { false };
     bool clean { false };
     bool binomial { false };
+    bool do_not_auto_update_params { false };
     uint32_t max_covg { 600 };
+    float min_absolute_gene_coverage { 3.0 };
+    float min_relative_gene_coverage { 0.05 };
+    float max_relative_gene_coverage { 10 };
```

Comment:
we define the same 3 vars to the same defaults in 2 files. Feels to me like we're going to accidentally modify a default value in one place and not the other, and end up confused. i think these consts should be the same for discover and compare?